openslides/motions/config_variables.py

```diff
@@ -90,11 +88,19 @@ def get_config_variables():
         name='motions_amendments_prefix',
         default_value='A',
         label='Prefix for the identifier for amendments',
-        hidden=True,
         weight=340,
         group='Motions',
         subgroup='Amendments')
 
+    yield ConfigVariable(
+        name='motions_amendments_apply_title_text',
+        default_value=False,
+        input_type='boolean',
+        label='Apply title and text for new amendments',
```

Comment:
You have to add this string to motions/site.js (ath the end of this file all motion config options are marked for translations)